dask/blockwise.py

```diff
@@ -393,10 +393,8 @@ def __dask_distributed_pack__(
             if isinstance(input_map, tuple):
                 # Use the `__dask_distributed_pack__` definition for the
                 # specified `BlockwiseIODeps` subclass
-                io_dep_map = getattr(
-                    import_allowed_module(input_map[0]),
-                    input_map[1],
-                )
```

Comment:
For this PR, the only changes in `blockwise.py` are intended to make the change [suggested in this comment](https://github.com/dask/dask/pull/7381#discussion_r595567232) (use fully qualified name of `BlockwiseIODeps` subclass, rather than a `(<module>, <name>)` tuple).